test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -629,6 +629,18 @@ test('mouse event buttons properties', function () {
     });
 });
 
+if (!browserUtils.isIE) {
+    test('timestamp', function () {
+        var options = { timeStamp: 'timestamp' };
```

Comment:
`timeStamp` is value in milliseconds.
So, let test value be in milliseconds too.
```js
var options = { timeStamp: 1000 };
```